pipelines/deepCoaddAssocSources.yaml

```diff
@@ -0,0 +1,13 @@
+description: |
```

Comment:
This file is called `deepCoaddAssocSources.yaml`, but does it really have anything to do with a `deepCoadd`? Should it be called something like `diaObjectAssociatedSources.yaml` or something similar instead?